neo/Network/P2P/TaskManager.cs

```diff
@@ -182,19 +182,40 @@ private void RequestTasks(TaskSession session)
                 session.Tasks[UInt256.Zero] = DateTime.UtcNow;
                 session.RemoteNode.Tell(Message.Create("getheaders", GetBlocksPayload.Create(Blockchain.Singleton.CurrentHeaderHash)));
             }
-            else if (Blockchain.Singleton.Height < session.Version.StartHeight)
+            else if (Blockchain.Singleton.Height < Blockchain.Singleton.HeaderHeight)
```

Comment:
Doing so will add too much network load.

Reply:
Hi, @erikzhang.
There is comment above, this network will probably not affect nodes that are already synced, it will just increase loads between sessions.
On the other hand, other type of mechanism could be designed for updating `session.Version.StartHeight` more periodically.

I am afraid that nowadays we are already asking for 500 blocks: https://github.com/neo-project/neo/blob/60a02eedc156a66e3955340f0024026f32d296f6/neo/Network/P2P/ProtocolHandler.cs#L159

With the other changes of this PR we limit is to `HashStart` to `HastStop`

Reply:
But if it is accepted it should go TestNet first. I do not know the impacts if nodes bypass messages.

Reply:
And, in fact, the load of request should not increase, @erikzhang, because we are just talking about an additional `hashtostop` that is added in this PR.
It would increase in the sense of requesting blocks for nodes that are behind yours. But, as I said, if they are behind no problem....they will discard the message. They are lagged anyway.

Reply:
@jsolman gave some tips and he will try on testnet when he has some free time.